tests/rules/data/syntax-error-in-dibi-prepared-statement.php

```diff
@@ -41,6 +41,13 @@ public function testPlaceholders(\Dibi\Connection $conn)
         $conn->fetchPairs('SELECT email FROM ada where email = %s', 'email@github.com');
     }
 
+    public function testIgnoredPlaceholders(\Dibi\Connection $conn)
+    {
+        $conn->query('SELECT %n, %n from %n', 'email', 'dadid', 'ada');
+        $conn->query('SELECT email, adaid FROM ada group by %by', 'email');
+        $conn->query('SELECT email, adaid FROM ada where email = %bin group by %by', 1, 'email');
```

Comment:
please add a test which contains placeholders and a syntax error, so we can see how this scenario is supposed to work

Reply:
added